frame/contracts/src/wasm/runtime.rs

```diff
@@ -393,6 +393,50 @@ define_env!(Env, <E: Ext>,
 		}
 	},
 
+	// Transfer some value to another account.
+	//
+	// If the value transfer was succesful zero is returned. Otherwise a non-zero value is returned.
+	// The scratch buffer is not touched. The receiver can be a plain account or a contract.
+	//
+	// - account_ptr: a pointer to the address of the beneficiary account
+	//   Should be decodable as an `T::AccountId`. Traps otherwise.
+	// - account_len: length of the address buffer.
+	// - value_ptr: a pointer to the buffer with value, how much value to send.
+	//   Should be decodable as a `T::Balance`. Traps otherwise.
+	// - value_len: length of the value buffer.
+	ext_transfer(
+		ctx,
+		account_ptr: u32,
+		account_len: u32,
+		value_ptr: u32,
+		value_len: u32
```

Comment:
```suggestion
		value_len: u32,
```

Reply:
I reverted that change because `define_env!` does not allow trailing comma.